### PR TITLE
[WJ-874] Change FTML web components to be autonomous, refactor styling

### PR DESCRIPTION
**Pull request by Monkatraz**
PR adjusts FTML web components and their styling as per the changes in #485. "Theming" related styling has been moved to a `theme/` folder.

### CODE REVIEW

File: client/modules/ftml-components/src/components/footnotes/_footnotes.scss [hunk 1]
```diff
@@ -1,102 +1,72 @@
...
-      padding-right: 0.5em;
-    }
+.wj-footnote-list-item-marker {
+  display: block;
+  cursor: pointer;
+}
 
-    .wj-footnote-sep {
-    }
+// .wj-footnote-sep {}
```
Comment (emmiegit) [inline client/modules/ftml-components/src/components/footnotes/_footnotes.scss]: Commented code

File: client/modules/ftml-components/src/components/math/_math.scss [hunk 2]
```diff
@@ -1,84 +1,66 @@
...
-    cursor: pointer;
-    user-select: none;
+.wj-math-ml.wj-math-ml-polyfilled math {
+  @include hide-visually;
+}
 
-    &.is-no-equation {
-      color: #b00;
-    }
+// .wj-equation-ref {}
```
Comment (emmiegit) [inline client/modules/ftml-components/src/components/math/_math.scss]: Commented code

File: client/modules/ftml-components/src/components/tabview/_tabview.scss [hunk 3]
```diff
@@ -2,70 +2,32 @@
...
+  flex-grow: 1;
+  max-width: 100%;
+  cursor: pointer;
+  user-select: none;
+}
 
-    &:active {
-      filter: brightness(0.9);
-    }
+// .wj-tabs-panel-list {}
```
Comment (emmiegit) [inline client/modules/ftml-components/src/components/tabview/_tabview.scss]: Ditto

**Review by emmiegit — APPROVED** [review]

File: client/modules/ftml-components/src/components/footnotes/_footnotes.scss (hunk 1 above)
Comment (Monkatraz) [reply client/modules/ftml-components/src/components/footnotes/_footnotes.scss]: I left these as comments in order to silence Stylelint errors, but also to still leave the intent of the selector being there intact. However, I can probably just remove them.

**Comment by Monkatraz** [comment]
thanks @ammongit

**Pull request merged**